src/app/pages/quickMaps/pages/baselineDetails/foodItems/foodItems.component.ts

```diff
@@ -19,51 +10,33 @@ import { CurrentDataService } from 'src/app/services/currentData.service';
 import { QuickMapsService } from '../../../quickMaps.service';
 import 'chartjs-chart-treemap';
 import { ChartData, ChartDataSets, ChartPoint, ChartTooltipItem } from 'chart.js';
-import { Subscription } from 'rxjs';
-import { GridsterItem } from 'angular-gridster2';
 import { ChartJSObject } from 'src/app/apiAndObjects/objects/misc/chartjsObject';
 import { DialogService } from 'src/app/components/dialogs/dialog.service';
 @Component({
   selector: 'app-food-items',
   templateUrl: './foodItems.component.html',
   styleUrls: ['./foodItems.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class FoodItemsComponent implements OnInit, OnDestroy {
+export class FoodItemsComponent implements OnInit {
   @ViewChild(MatPaginator, { static: true }) paginator: MatPaginator;
   @ViewChild(MatSort) sort: MatSort;
-  @Input()
```

Comment:
I'm not sure that you really needed to remove a lot of these lines of code in this file.  I think they're needed.  (Although it doesn't look like it was complete in the first place).

Reply:
true - the ondestroy is needed, the function seems to have been removed:

```
ngOnDestroy(): void {
    this.resizeSub.unsubscribe();
  }
```

Reply:
Ah, I had a bit of a conflict resolution issue with that file. Will update accordingly